packages/libs/lambda-at-edge/package.json

```diff
@@ -63,6 +63,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "1.0.0-rc.3",
     "@hapi/accept": "5.0.1",
+    "@sls-next/routing": "link:../routing",
```

Comment:
Maybe call this @sls-next/core as it might have other logic/utilities besides routing logic